packages/babel-helper-get-function-arity/src/index.ts

```diff
@@ -1,7 +1,8 @@
 import * as t from "@babel/types";
+import type { Function } from "@babel/types";
 
-export default function (node): number {
-  const params: Array<Object> = node.params;
+export default function (node: Function): number {
```

Comment:
Can this be `t.Function`? Seeing `Function` is a bit confusing, because of the built-in `Function` type.